<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-OB6MLR4" ignored="0" name="netBankingApp Test" tests="7" failures="3" timestamp="27 Sep 2020 13:30:54 GMT" time="125.001" errors="0">
  <testcase name="loginTest" time="7.547" classname="com.netBankingApp.TestCases.TC_LoginTest_001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.netBankingApp.TestCases.TC_LoginTest_001.loginTest(TC_LoginTest_001.java:37)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="LoginDDT" time="12.159" classname="com.netBankingApp.TestCases.TC_LoginDDT_002"/>
  <testcase name="LoginDDT" time="4.827" classname="com.netBankingApp.TestCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.netBankingApp.TestCases.TC_LoginDDT_002.LoginDDT(TC_LoginDDT_002.java:30)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- LoginDDT -->
  <testcase name="LoginDDT" time="10.52" classname="com.netBankingApp.TestCases.TC_LoginDDT_002"/>
  <testcase name="LoginDDT" time="5.248" classname="com.netBankingApp.TestCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.netBankingApp.TestCases.TC_LoginDDT_002.LoginDDT(TC_LoginDDT_002.java:30)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- LoginDDT -->
  <testcase name="LoginDDT" time="11.329" classname="com.netBankingApp.TestCases.TC_LoginDDT_002"/>
  <testcase name="addNewCustomer" time="13.818" classname="com.netBankingApp.TestCases.TC_AddCustomerTest_003"/>
</testsuite> <!-- netBankingApp Test -->
